### PR TITLE
[EventHubs] Sample, Doc, Test improvement

### PR DESCRIPTION
**Pull request by yunhaoling**
Revert back to the original directory and improve the code snippet scripts

### CODE REVIEW

File: sdk/eventhub/azure-eventhub/README.md [hunk 1]
```diff
@@ -284,7 +284,7 @@ connection_str = '<< CONNECTION STRING FOR THE EVENT HUBS NAMESPACE >>'
 consumer_group = '<< CONSUMER GROUP >>'
 eventhub_name = '<< NAME OF THE EVENT HUB >>'
 storage_connection_str = '<< CONNECTION STRING FOR THE STORAGE >>'
-container_name = '<<STRING FOR THE BLOB NAME>>'
+container_name = '<<STRING FOR THE BLOB CONTAINER NAME>>'
```
Comment (YijunXieMS) [inline sdk/eventhub/azure-eventhub/README.md]: <<NAME OF THE BLOB CONTAINER>>>?
People will understand NAME is a string.

File: sdk/eventhub/azure-eventhub/azure/eventhub/_producer_client.py [hunk 2]
```diff
@@ -232,12 +232,13 @@ def create_batch(self, **kwargs):
          will assign to all partitions using round-robin.
         :keyword str partition_key: With the given partition_key, event data will be sent to
          a particular partition of the Event Hub decided by the service.
-        :keyword int max_size_in_bytes: The maximum size of bytes data that an EventDataBatch object can hold.
+        :keyword int max_size_in_bytes: The maximum size of bytes data that an EventDataBatch object can hold. By
+         default the value is determined by the Event Hubs service.
```
Comment (YijunXieMS) [inline sdk/eventhub/azure-eventhub/azure/eventhub/_producer_client.py]: ...  by the Event Hubs service and your price tier

File: sdk/eventhub/azure-eventhub/README.md (hunk 1 above)
Comment (yunhaoling) [reply sdk/eventhub/azure-eventhub/README.md]: updated

File: sdk/eventhub/azure-eventhub/azure/eventhub/_producer_client.py (hunk 2 above)
Comment (yunhaoling) [reply sdk/eventhub/azure-eventhub/azure/eventhub/_producer_client.py]: updated, I just use the word `Event Hubs tier`

**Pull request merged**